data/systemd/snapd-user.service.in

```diff
@@ -0,0 +1,10 @@
+[Unit]
```

Comment:
They don't have to. @ basically allows to pass arguments to the systemd unit which we don't need in this case as systemd will ensure we're started as the right user and in the right scope.